ui/raidboss/raidboss.ts

```diff
@@ -1,16 +1,13 @@
 import { addOverlayListener } from '../../resources/overlay_plugin_api';
-
-import './raidboss_config';
-import '../../resources/timerbar';
-
-import { PopupText, PopupTextGenerator } from './popup-text';
-import { TimelineController, TimelineLoader, TimelineUI } from './timeline';
-
-import UserConfig from '../../resources/user_config';
 import { addRemotePlayerSelectUI } from '../../resources/player_override';
+import UserConfig from '../../resources/user_config';
 import raidbossFileData from './data/raidboss_manifest.txt';
-
+import { PopupText, PopupTextGenerator } from './popup-text';
 import Options from './raidboss_options';
+import { TimelineController, TimelineLoader, TimelineUI } from './timeline';
+
```

Comment:
Do these two below blocks of import not count for newline-between: never?

Reply:
Ah, so no; this plugin never touches "imports with side effects"; although they do need to be clumped together. I went with putting them below, following the import order above and putting *script before css/assets.

Open to thoughts

Reply:
Ah no, mostly was just curious how that worked.  It makes sense not to resort those, I guess.